snmp/datadog_checks/snmp/snmp.py

```diff
@@ -517,10 +517,13 @@ def submit_metric(self, name, snmp_value, forced_type, tags):
             self.log.warning('No such Mib available: %s', name)
             return
 
-        metric_name = self.normalize(name, prefix='snmp')
+        if 'metric_suffix' in options:
+            metric_name = self.normalize('{}.{}'.format(name, options['metric_suffix']), prefix='snmp')
+        else:
+            metric_name = self.normalize(name, prefix='snmp')
```

Comment:
(nit) avoid duplication of `self.normalize`
```suggestion
        if 'metric_suffix' in options:
            name = '{}.{}'.format(name, options['metric_suffix'])
        metric_name = self.normalize(name, prefix='snmp')
```

Reply:
That changes the logging below